src/main/resources/config/en.yml

```diff
@@ -16,6 +16,8 @@ Channels: {"global": "000000000000000000"}
 # Console channel numerical ID (NOT NAME), leave blank to disable the console channel all together
 DiscordConsoleChannelId: "000000000000000000"
 
+DiscordInviteLink: ""
```

Comment:
Move to messages.yml below/near `DiscordCommandFormat`, add a comment to the option, default value of `discord.gg/changethisinthemessages.yml` (or something along those lines)

Reply:
No problem with the changing of the default value, but moving it to the messages.yml file removes the benefit of this patch, e.g. to separate config from messages.

Reply:
The benafit is that the option can be changed for 2 options at once, that is still there. Putting it in a entirely different place doesn't make sense to me, and putting it near the Discord command format makes sense; most people configure that, unlike require linked 4 gameplay that is used by ~38 servers out of 2000+ on record (https://bstats.org/plugin/bukkit/DiscordSRV/387)

Reply:
Being able to exchange messages.yml files with others, having tailored the language to tailor the style, without it containing server related configuration and settings is the goal of this. De-duplication is an added bonus.

Reply:
Ok, that's a valid point